source/get-started.txt

```diff
@@ -0,0 +1,49 @@
+.. _ruby-get-started:
+
+================================
+Get Started with the Ruby Driver
+================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
+ 
+.. meta::
+   :description: Learn how to create an app to connect to MongoDB deployment by using the Ruby driver.
+   :keywords: quick start, tutorial, basics
+
+.. toctree::
+
+   Download & Install </get-started/download-and-install/>
+
+.. TODO:
+   Create a Deployment </get-started/create-a-deployment>
+   Create a Connection String </get-started/create-a-connection-string>
+   Connect to MongoDB </get-started/connect-to-mongodb>
+   Next Steps </get-started/next-steps>
+
+Overview
+--------
+
+The {+driver-long+} is a library that allows Ruby applications to interact with
+MongoDB databases. You can use the {+driver-short+} to connect to MongoDB and perform
+common data operations. This guide shows you how to create an application that uses the
+{+driver-short+} to connect to a MongoDB cluster hosted on MongoDB Atlas
+and query data in your cluster.
+
+.. tip:: 
+
+   MongoDB Atlas is a fully managed cloud database service that hosts your MongoDB
+   deployments. You can create your own free (no credit card required) MongoDB Atlas 
+   deployment by following the steps in this guide.
+
+Follow this guide to connect a sample Ruby application to a MongoDB Atlas
+deployment. If you prefer to connect to MongoDB using a different driver or
```

Comment:
Is this first sentence needed after the last sentence in the previous (non-tip) paragraph?

Reply:
It seems repetitive, I'll remove it